assets/js/trento.jsx

```diff
@@ -1,13 +1,17 @@
 import React from 'react';
 import { createRoot } from 'react-dom/client';
 
-import { Routes, Route, BrowserRouter } from 'react-router-dom';
+import {
+  createRoutesFromElements,
+  createBrowserRouter,
+  Route,
+  RouterProvider,
+  Outlet,
+} from 'react-router-dom';
 
 import { Toaster } from 'react-hot-toast';
 import { Provider } from 'react-redux';
 
-import { ErrorBoundary } from 'react-error-boundary';
```

Comment:
I'm just removing the `react-error-boundary` usage, as it can go in the router code itself.
Besides that, we were not using the `onReset` callback, as the `resetErrorBoundary` is not called anywhere.